modules/sku/src/ProductInfoRequestedEvent.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+namespace Drupal\acm_sku;
+
+use Drupal\acm_sku\Entity\SKUInterface;
+use Symfony\Component\EventDispatcher\Event;
+
+/**
+ * Class ProductInfoRequestedEvent.
+ *
+ * @package Drupal\acm_sku
+ */
+class ProductInfoRequestedEvent extends Event {
+
+  const EVENT_NAME = 'product_info_requested';
```

Comment:
All looks fine except this as, as per D8 all events should be defined as an abstract class (for documentation purpose). I know we already have few events but this can be starting points (for documentation purpose as well).

Reply:
can you share an example? quick search for `extends Event` in all files didn't return me any class with interface 

for instanc e`class ConfigImporterEvent extends Event`, doesn't have an interface.

Reply:
https://api.drupal.org/api/drupal/core%21core.api.php/group/events/8.6.x

And see https://api.drupal.org/api/drupal/vendor%21symfony%21http-kernel%21KernelEvents.php/class/KernelEvents/8.6.x for example

Reply:
@joshirohit100 thanks for the idea, it also allowed me to add some documentation on how to use this event and service.